subsys/shell/modules/kernel_service.c

```diff
@@ -218,6 +219,17 @@ static int cmd_kernel_stacks(const struct shell *shell,
 }
 #endif
 
+static int cmd_kernel_sleep(const struct shell *sh,
```

Comment:
I know `sh` is different from `shell` used in the other commands, but coccinelle was whining about this rule https://gitlab.com/MISRA/MISRA-C/MISRA-C-2012/Example-Suite/-/blob/master/R_05_07.c

Reply:
Renaming `*shell`=>`*sh` is a fine approach (used that as well).